CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,7 @@
 
 - [Backup reports] Fix backup report not sent in case of interrupted backup job (PR [#4772](https://github.com/vatesfr/xen-orchestra/pull/4772))
 - Fix TLS error (`unsupported protocol`) with XenServer <= 6.5 and Node >= 12 (PR [#8437](https://github.com/vatesfr/xen-orchestra/pull/8437))
+- [Metadata backup] fix metadata backup timeout [#4819](https://github.com/vatesfr/xen-orchestra/issues/4819) (PR [#4831](https://github.com/vatesfr/xen-orchestra/pull/4831))
```

Comment:
Not good enough, what potential user problemes have you solved with this PR?

Reply:
This PR is created to fix a client issue which has a timeout on his metadata backup. But, it can also resolves other issues linked to different backups which aren't reported yet.

Reply:
Indeed, it solves backup issues.

Which can happen in which situation?

Reply:
If the number of backups executed in parallel exceeds the current limitation (10)

Reply:
```suggestion
- [Metadata backup] Fix timeout when lots of pools are backed up [#4819](https://github.com/vatesfr/xen-orchestra/issues/4819) (PR [#4831](https://github.com/vatesfr/xen-orchestra/pull/4831))
```